scripts/test_tag

```diff
@@ -6,18 +6,18 @@
 # The workflow for that is to edit `.github/workflows/schemacode_ci.yml` and add a secondary branch name to the `on` section right after master.
 # This will allow you to keep pushing to the branch as if it were master and unix-time tagged archives being uploaded to test.pypi.
 
-VERSION_FILE="src/schema/SCHEMA_VERSION"
+DATETIME="$(date +%Y%m%d%H%M)"
 
-pushd ..
-	export MY_BIDS_DATE="$(date +%s)"
-	REAL_VERSION=$(cat "${VERSION_FILE}")
-	DUMMY_VERSION="0.0.${MY_BIDS_DATE}-dev"
-	echo "${DUMMY_VERSION}" > "${VERSION_FILE}"
-	git commit -a -m "debug"
-	git tag -a "schema-$DUMMY_VERSION" -m "debug"
-	git push origin
-	git push origin "schema-$DUMMY_VERSION"
```

Comment:
Removed automatic pushes...

Reply:
I think this is handy, but I can add an option for push later.